### PR TITLE
Guessable letters field 

### PR DESCRIPTION
**Pull request by lisapan**
- Guessable letters field
- Can't guess same letter twice
- Can't draw parts after full word guessed
- Cant guess more letters after losing
- Only letter keys can be pressed

### CODE REVIEW

File: HangMan.java [hunk 1]
```diff
@@ -49,47 +45,89 @@ public void start(Stage primaryStage) throws Exception {
...
-                        break;
-                    case 3 : drawArm("right");
-                        break;
-                    case 4 : drawArm("left");
-                        break;
-                    case 5 : drawBody();
-                        break;
-                    case 6 : drawHead();
-                        break;
+            if (alphabet.indexOf(guess) == -1) { return; }
```
Comment (jraffileTOAST) [inline HangMan.java]: Why have this in one line and not the other?  It works.  But, js.

File: HangMan.java [hunk 2]
```diff
@@ -140,5 +178,7 @@ public char getLetter(){
             return this.letter;
         }
     }
+
+
```
Comment (jraffileTOAST) [inline HangMan.java:218]: I like this one.

**Review by jraffileTOAST — APPROVED** [review]
I like it!

File: HangMan.java [hunk 3]
```diff
@@ -21,6 +20,9 @@
     int MAX_GUESSES = 6;
     int guessesLeft = MAX_GUESSES;
     ArrayList<HangmanTile> wordTiles = new ArrayList<HangmanTile>();
+    ArrayList<HangmanTile> alphabetTiles = new ArrayList<HangmanTile>();
+    ArrayList<Character> alphabet = new ArrayList<Character>();
+
```
Comment (ccampbell08) [inline HangMan.java:26]: derp

**Pull request merged**